### PR TITLE
Add quick links section to README

### PR DESCRIPTION
**Pull request by bhancock8**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,17 +7,23 @@
...
 
 
-## [Snorkel webpage](http://snorkel.stanford.edu)
+# Quick Links
+* [Snorkel website](snorkel.org)
+* [Snorkel tutorials](snorkel-team/snorkel-tutorials)
+* [Snorkel documentation](https://snorkel.readthedocs.io/en/master/)
+* [Snorkel community forum](https://spectrum.chat/snorkel)
+* [Snorkel mailing list](https://groups.google.com/forum/#!forum/snorkel-ml)
+* [Snorkel twitter](https://twitter.com/SnorkelML)
```
Comment (vincentschen) [inline README.md]: maybe add a short section under Community to follow us on Twitter + use #snorkelML ?

Comment (henryre) [reply README.md]: Twitter capitalized

File: README.md [hunk 2]
```diff
@@ -27,21 +39,30 @@ pip3 install -U .
...
 
-If you want to set up a development environment in order to contribute back to Snorkel,
-see our [Contributing doc](./CONTRIBUTING.md).
 
+# Discussion
 
-## Issues and feature requests
-We use [GitHub Issues](https://github.com/snorkel-team/snorkel/issues) as a place to post bugs and feature requests — anything code-related.
+## Issues
+We use [GitHub Issues](https://github.com/snorkel-team/snorkel/issues) for posting bugs and feature requests — anything code-related.
```
Comment (henryre) [inline README.md]: nit: training ws

File: README.md [hunk 3]
```diff
@@ -6,18 +6,30 @@
 [![style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/python/black)
 [![license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+Programmatically Building & Managing Training Data 
 
-## [Snorkel webpage](http://snorkel.stanford.edu)
+# Quick Links
+* [Snorkel website](http://snorkel.org)
+* [Snorkel tutorials](https://github.com/snorkel-team/snorkel-tutorials)
+* [Snorkel documentation](https://snorkel.readthedocs.io/en/master/)
```
Comment (henryre) [inline README.md]: drop the /en/master...want it to go to whatever we set as the default

File: README.md (hunk 2 above)
Comment (henryre) [reply README.md]: trailing

**Pull request merged**